lua/telescope/mappings.lua

```diff
@@ -30,6 +30,7 @@ mappings.default_mappings = config.values.default_mappings
       ["<C-q>"] = actions.send_to_qflist + actions.open_qflist,
       ["<M-q>"] = actions.send_selected_to_qflist + actions.open_qflist,
       ["<C-l>"] = actions.complete_tag,
+      ["<C-z>"] = actions._toggle_registered_actions,
```

Comment:
I don't think that this is a good choice of mapping for this, as it is already used by neovim to "suspend". Could we use a mapping involving `?`?

Reply:
Default mapping will be removed once PR is final. `<C-?>` eg cannot be mapped because, if I'm not mistaken, the terminal doesn't send the keycodes you'd expect. If we'd want a default mapping, could you try if `<C-_>` does what you'd expect? Might be my German keyboard though which is set to US layout.

Reply:
we can just make it normal mode "?" probably, right? That seems like a good mapping.

Reply:
I guess discouraged since -
- we currently mirror normal/insert mode mappings for any built-in mapping (and insert mode is default for most users.. a bit unergonomic to switch to normal mode for a built-in action)
- we default to `only_show_current_mode = true` here as mappings are duplicated in float across modes b/c of mirroring; I guess we could also merge them in "n/i" -> "..." -> "...", which would look inconsistent with custom actions though

I'm only the devils advocate here though :) happy to just map it to "?" in normal mode